test/check-machines-create

```diff
@@ -2787,6 +2787,25 @@ vnc_password= "{vnc_passwd}"
             b.wait_text(f"#vm-{vmName}-system-state", "Running")
             b.wait_not_present(f"#vm-{vmName}-system-state-error")
 
+    def testNoSystemOptionOnLimitedAccess(self):
+        b = self.browser
+
+        self.login_and_go("/machines")
+        self.waitPageInit()
+
+        b.click("#create-new-vm")
+        b.wait_visible("#create-vm-dialog")
+        b.wait_visible("#connectionName-system")
+        b.click("#create-vm-dialog .pf-v5-c-button.pf-m-link")
+        b.wait_not_present("#create-vm-dialog")
+
+        b.drop_superuser()
+        self.waitPageInit()
+
+        b.click("#create-new-vm")
+        b.wait_visible("#create-vm-dialog")
+        b.wait_not_present("#connection")
```

Comment:
Do we have any other test which covers VM creation in unprivileged mode? If so, this would better fit there. If not, that's a major gap indeed, and this test should actually create the VM and check that it starts and boots fine (including the "wait for login prompt").

Reply:
I believe this is confusingly what TestMachinesLifecycle does:

```
    def _testBasic(self, user=None, superuser=True, expect_empty_list=False, run_pixel_tests=False):
        b = self.browser
        m = self.machine

        # We want no initial watchdog
        args = self.createVm("subVmTest1", os="linux2016")

        self.login_and_go("/machines", user=user, superuser=superuser)
```

So we can drop this test